CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
If we use Conventional Commits, we could auto generate the changelog (ex: https://medium.com/jobtome-engineering/how-to-generate-changelog-using-conventional-commits-10be40f5826c) WDYT?

Reply:
As you used Conventional Commits, I also suggest mentioning it somewhere. Readme? Changelog? https://www.conventionalcommits.org/en/v1.0.0-beta.2/

Reply:
I tried out standard-version (https://github.com/conventional-changelog/standard-version) but i'm struggling to make it work. While researching, i found out that standard-release is deprecated (https://www.npmjs.com/package/standard-version#bumpfiles-packagefiles-and-updaters) and npm recommends using release-please instead (https://github.com/googleapis/release-please). release-please seems to be creating release PRs. I think i should wait until this PR is resolved before opening a new one with release-please, WDYT ?

Reply:
I did update both README and CHANGELOG files to mention conventional commits (see commit d5756db8504d30a2b43a366a3dd13af8da0abbe3).